console.py

```diff
@@ -160,5 +160,18 @@ def do_update(self, arg):
         else:
             print("** class doesn't exist **")
 
+    def do_count(self, args):
+        """Count current number of class instances"""
+        count = 0
+        __objects = storage.all()
```

Comment:
Any reason we are creating a private variable? It should already only be accessible inside this function. Also, why not have this set to `storage.all().items()` right of the bat? We never use `__objects` outside of the following `for` loop, so there's no point retaining the dictionary.

Reply:
Actually I think we can just use our new method storage.count() - let me change that real quick!